### PR TITLE
rav1e 0.6.2

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

~`resource` blocks may require updates.~

### CODE REVIEW

**Comment by chenrui333** [comment]
vice test failure on `12-arm`, also seen in #118935

**Comment by chenrui333** [comment]
qt bottle issue on linux build (relates to #117089)
```
  Missing libraries:
    unexpected (libre2.so.9)
```

**Comment by cho-m** [comment]
> qt bottle issue on linux build (relates to #117089)
> 
> ```
>   Missing libraries:
>     unexpected (libre2.so.9)
> ```

That means we needed to revision bump or rebuild `qt` as it is linked to `re2` and needs to link to `libre2.so.10` now. In that PR, we also lost the `pyqt` and `pyqt@5` Linux bottles and need to recover those.

**Comment by iMichka** [comment]
Needs #119257 and #119090

**Review by SMillerDev — APPROVED** [review]

File: Formula/rav1e.rb [hunk 1]
```diff
@@ -5,12 +5,13 @@ class Rav1e < Formula
   head "https://github.com/xiph/rav1e.git", branch: "master"
 
   stable do
-    url "https://github.com/xiph/rav1e/archive/v0.6.1.tar.gz"
-    sha256 "dd12132ad9dac229ce00a9caad132c4ad23d7db2b3ad4b5a59e89658fee04d9a"
+    url "https://github.com/xiph/rav1e/archive/v0.6.2.tar.gz"
+    sha256 "8fe8d80bc80a05ee33113c0ee19779d9c57189e5434c8e1da8f67832461aa089"
 
+    # keep the version in sync
     resource "Cargo.lock" do
```
Comment (SMillerDev) [inline Formula/rav1e.rb:12]: I think this would be an excelent one for @nandahkrishna their recent livecheck improvements

**Pull request closed without merging**